packages/components/molecules/f-restaurant-card/src/components/subcomponents/IconText.vue

```diff
@@ -86,6 +86,10 @@ export default {
     color: $color-green;
 }
 
+.c-restaurantCard-iconText-content--blue {
+    color: $color-blue;
+}
+
```

Comment:
`$color-blue` is a global pie token, catching up with Ash last week he highlighted that we should avoid referencing the global token directly but instead use the alias tokens. Maybe this could be updated to `$color-support-info`

Attached is the pie token link if that's useful - https://unpkg.com/browse/@justeat/pie-design-tokens@1.4.0/dist/jet.scss

Also I would revisit the naming of this `c-restaurantCard-iconText-content--blue` to be more descriptive of what the different state is rather than what it looks like to make it more future proof.  So maybe that's `content--info` or something on those lines

Reply:
good point - updated!